observer/probers/http/http.go

```diff
@@ -12,11 +13,17 @@ type HTTPProbe struct {
 	url       string
 	rcodes    []int
 	useragent string
+	insecure  bool
 }
 
 // Name returns a string that uniquely identifies the monitor.
+
 func (p HTTPProbe) Name() string {
-	return fmt.Sprintf("%s-%d-%s", p.url, p.rcodes, p.useragent)
+	s := fmt.Sprintf("%s-%d-%s", p.url, p.rcodes, p.useragent)
+	if p.insecure {
+		return s + "-insecure"
+	}
+	return s
```

Comment:
I just realized that a slightly-more-idiomatic way to do this might be:
```suggestion
	insecure := ""
	if p.insecure {
		insecure = "-insecure"
	}
	return fmt.Sprintf("%s-%d-%s%s", p.url, p.rcodes, p.useragent, insecure)
```

Doing it this way is absolutely not necessary, just pointing it out as an option for future reference.